test/TestMultiTag.cpp

```diff
@@ -171,6 +171,11 @@ void TestMultiTag::testReferences(){
     dt.addReference(da_2);
     CPPUNIT_ASSERT_MESSAGE(counterrmsg.str(), dt.referenceCount() == 2);
 
+    std::stringstream haserrmsg;
+    haserrmsg << "TestMultiTag::testReference: Has method did not work!";
```

Comment:
That was the old style before we switched to use the CPPUNIT_ASSERT without message. 
Actually we could simply ```` CPPUNIT_ASSERT_MESSAGE("some message", x == y)```` if we want to keep the messages. 
imho, we can live with that style.  I kind of like the messages but probably they are more work than they are worth.